orttraining/orttraining/core/optimizer/compute_optimizer/sceloss_compute_optimization.h

```diff
@@ -32,10 +32,10 @@ namespace onnxruntime {
  * 2. Its 2nd output (log_prob) MUST NOT be a graph output and MUST NOT be consumed by other nodes.
  * 3. Its ignore_index exists and is a constant scalar value.
  * 4. Its 2nd input label's input node is not a `ShrunkGather` node (to avoid this transformer duplicated applied).
- * 5. Its 2nd input label is 1) a graph input or 2) output of a Reshape node taking a graph input as its data input.
+ * 5. Following PythonOp (FlagAndPrintDensity).
```

Comment:
so this is correcting the original comment?

Reply:
yes, this is to update the implementation details.